### ISSUE TITLE
Check if Saved Link exists

### ISSUE THREAD

**Issue by dryanmedia** [INN/link-roundups#44]
Would it make sense to check if a URL already exists when adding new saved links?

Can see potential for why a user might want duplicate entries, but would be useful if the plugin prompted you to avoid duplicate links.

**Comment by aschweigert** [comment]
yeah, i mean i guess theoretically you might want to have different descriptions for the same link for different purposes...maybe. seems like kind of a weird edge case though.
